### PR TITLE
Integrates with AWS Lambda

### PR DESCRIPTION
**Pull request by AaronVasquez**

### CODE REVIEW

File: services/iota_wrappers.go [hunk 1]
```diff
@@ -102,7 +128,7 @@ func init() {
 		panic("Invalid IRI host: Check the .env file for HOST_IP")
 	}
 
-	provider := "http://" + host_ip + ":14265"
```
Comment (pzhao5) [inline services/iota_wrappers.go]: this will fail. revert

File: services/iota_wrappers.go [hunk 2]
```diff
@@ -352,15 +379,59 @@ func sendChunksToChannel(chunks []models.DataMap, channel *models.ChunkChannel)
 		models.DB.ValidateAndSave(&chunk)
 	}
 
-	channel.EstReadyTime = SetEstimatedReadyTime(Channel[channel.ChannelID], len(chunks))
-	models.DB.ValidateAndSave(channel)
+	if os.Getenv("ENABLE_LAMBDA") == "true" {
+		go func() {
+			// Batch chunks by limit
```
Comment (pzhao5) [inline services/iota_wrappers.go]: move this to a different function

File: services/iota_wrappers.go [hunk 3]
```diff
@@ -41,8 +44,7 @@ type IotaService struct {
 	FindTransactions
 	GetTransactionsToApprove
 }
-
-type ProcessingFrequency struct {
+type ProcessingFhttpRequency struct {
```
Comment (rfornea) [inline services/iota_wrappers.go]: Is `ProcessingFhttpRequency` a spelling error?

Comment (AaronVasquez) [reply services/iota_wrappers.go]: yeah, didnt mean to change that. Good catch

File: services/iota_wrappers.go [hunk 4]
```diff
@@ -352,15 +383,19 @@ func sendChunksToChannel(chunks []models.DataMap, channel *models.ChunkChannel)
 		models.DB.ValidateAndSave(&chunk)
 	}
 
-	channel.EstReadyTime = SetEstimatedReadyTime(Channel[channel.ChannelID], len(chunks))
-	models.DB.ValidateAndSave(channel)
+	if os.Getenv("ENABLE_LAMBDA") == "true" {
+		go batchPowOnLambda(chunks)
+	} else {
```
Comment (rfornea) [inline services/iota_wrappers.go]: The ProcessUnassignedChunks looks at the number of ready channels and that influences how many chunks it sends.  It also calls a method (in this file, I think) that tracks how long the broker is taking to complete PoW, which determines how often process_unassigned_chunks runs.  These are things we'll probably want to change, before we deploy the lambda changes to prod.  

(This comment isn't in reference to this particular PR, just a general comment)

**Pull request merged**